src/p2G4_main.c

```diff
@@ -169,10 +201,15 @@ static void f_rx_search(uint d) {
     return;
   }
 
-  tx_d = txl_find_fitting_tx(&rx_status->rx_s, current_time );
+  rx_queue_search_end(d);
+  return;
+}
+
+static void f_rx_found(uint d){
+  rx_status_t *rx_status = &rx_a[d];
 
-  if (tx_d >= 0) {
-    rx_status->tx_nbr = tx_d;
+  if (rx_status->tx_nbr >= 0) {
+    const uint tx_d = rx_status->tx_nbr;
```

Comment:
Curiosity question: Did you see any difference in the resulting code from adding this const?

Reply:
nope, just a const correctness habit. I'll remove it